docs/source/10_deployment/12_dask.md

```diff
@@ -0,0 +1,311 @@
+    """Run the pipeline."""
+    if parallel and runner:
+        raise KedroCliError(
+            "Both --parallel and --runner options cannot be used together. "
+            "Please use either --parallel or --runner."
+        )
+    runner = runner or "SequentialRunner"
+    if parallel:
+        runner = "ParallelRunner"
+
+    tag = _get_values_as_tuple(tag) if tag else tag
+    node_names = _get_values_as_tuple(node_names) if node_names else node_names
+
+    with KedroSession.create(env=env, extra_params=params) as session:
+        runner_instance = _instantiate_runner(runner, is_async, context)
+        session.run(
+            tags=tag,
+            runner=runner_instance,
+            node_names=node_names,
+            from_nodes=from_nodes,
+            to_nodes=to_nodes,
+            from_inputs=from_inputs,
+            load_versions=load_version,
+            pipeline_name=pipeline,
+        )
+```
+
+where the helper function `_instantiate_runner()` looks like this:
+
+```python
+def _instantiate_runner(runner, is_async, project_context):
+    runner_class = load_obj(runner, "kedro.runner")
+    runner_kwargs = dict(is_async=is_async)
+
+    if runner.endswith("DaskRunner"):
+        client_args = project_context.params.get("dask_client") or {}
+        runner_kwargs.update(client_args=client_args)
+
+    return runner_class(**runner_kwargs)
+```
+
+### Deploy
+
+You're now ready to trigger the run. Without any further configuration, the underlying Dask [`Client`](http://distributed.dask.org/en/stable/api.html#distributed.Client) creates a [`LocalCluster`](http://distributed.dask.org/en/stable/api.html#distributed.LocalCluster) in the background and connects to that:
+
+```bash
+kedro run --runner kedro_tutorial.runner.DaskRunner
+```
+
+#### Set up Dask and related configuration
+
+To connect to an existing Dask cluster, you'll need to set the Dask-related configuration that the runner will use. Create the `conf/dask/` directory and add a `parameters.yml` file inside of it with the following keys:
+
+```yaml
+dask_client:
+  address: 127.0.0.1:8786
+```
+
+Next, [set up scheduler and worker processes on your local computer](http://distributed.dask.org/en/stable/quickstart.html#setup-dask-distributed-the-hard-way):
+
+```bash
+$ dask-scheduler
+Scheduler started at 127.0.0.1:8786
+
+$ PYTHONPATH=$PWD/src dask-worker 127.0.0.1:8786
+$ PYTHONPATH=$PWD/src dask-worker 127.0.0.1:8786
+$ PYTHONPATH=$PWD/src dask-worker 127.0.0.1:8786
+```
+
+```eval_rst
+.. note::  The above code snippet assumes each worker is started from the root directory of the Kedro project in a Python environment where all required dependencies are installed.
+```
+
+You're once again ready to trigger the run. Execute the following command:
+
+```bash
+kedro run --env dask --runner kedro_tutorial.runner.DaskRunner
+```
+
+You should start seeing tasks appearing on [Dask's diagnostics dashboard](http://127.0.0.1:8787/status):
```

Comment:
I'm guessing this needs to be added to `linkcheck_ignore`, maybe once the build runs lol.